src/Files.App/Data/Models/ItemViewModel.cs

```diff
@@ -1642,8 +1642,17 @@ private async Task<int> EnumerateItemsFromStandardFolderAsync(string path, Cance
 					});
 
 					rootFolder ??= await FilesystemTasks.Wrap(() => StorageFileExtensions.DangerousGetFolderFromPathAsync(path));
-					if (rootFolder?.DisplayName is not null)
-						currentFolder.ItemNameRaw = rootFolder.DisplayName;
+					if (rootFolder is not null)
+					{
+						if (rootFolder.DisplayName is not null)
+							currentFolder.ItemNameRaw = rootFolder.DisplayName;
+
+						if (!string.Equals(path, Constants.UserEnvironmentPaths.RecycleBinPath, StringComparison.OrdinalIgnoreCase))
+						{
+							var syncStatus = await CheckCloudDriveSyncStatusAsync(rootFolder);
```

Comment:
What's the sync status needed for?

Reply:
By default all folders are marked as `FolderOnline` (that is the 0 value of the enum). We don't get the size properties for online folders, and that was causing the issue

Reply:
Good catch!